internal/runners/branch/add.go

```diff
@@ -0,0 +1,59 @@
+package branch
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+type Add struct {
+	out     output.Outputer
+	project *project.Project
+}
+
+type AddParams struct {
+	Label string
+}
+
+func NewAdd(prime primeable) *Add {
+	return &Add{
+		out:     prime.Output(),
+		project: prime.Project(),
+	}
+}
+
+func (a *Add) Run(params AddParams) error {
+	logging.Debug("ExecuteAdd")
+
+	project, err := model.FetchProjectByName(a.project.Owner(), a.project.Name())
+	if err != nil {
+		return locale.WrapError(err, "branch_list_proejct_err", "Could not get project details for project: {{.V0}}", a.project.Namespace().String())
+	}
+
+	branchID, err := model.AddBranch(project.ProjectID, params.Label)
+	if err != nil {
+		return locale.WrapError(err, "err_add_branch", "Could not add branch")
```

Comment:
We should try to assert here if this is because the branch label is already used. Also can you make sure that the error from the API bubbles up here?

Reply:
Confirmed that the API error bubbles up here. The output is below and seems fairly clear. Do we want to assert the error here so we can add more context to the error? 

```
╔═══════════════╗
║ Adding branch ║
╚═══════════════╝

Something Went Wrong
────────────────────
 x Could not add branch
 x This project already contains a branch with that name

Need More Help?
───────────────
 • Run → `state add --help` for general help
 • Visit the Forum → https://community.activestate.com/c/state-tool/
```